google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -290,6 +296,29 @@ public EnhancedBigtableStub(EnhancedBigtableStubSettings settings, ClientContext
     pingAndWarmCallable = createPingAndWarmCallable();
   }
 
+  @VisibleForTesting
+  public EnhancedBigtableStub(EnhancedBigtableStubSettings settings, ClientContext clientContext, RateLimitingStats rateLimitingStats) {
```

Comment:
If you move the RateLimitingCallable and RateLimitingCallableTests under stub/ you won't need to make this class public.